moco/src/main/java/com/cognifide/aem/stubs/moco/MocoStubs.java

```diff
@@ -83,11 +98,25 @@ protected void deactivate() {
 
   private void start() {
     LOG.info("Starting AEM Stubs Moco Server");
+
     if (config.logging()) {
-      server = httpServer(config.port(), ApiUtils.log(LOG::info));
+      server = (ActualHttpServer) httpServer(config.port(), ApiUtils.log(LOG::info));
```

Comment:
why not ```MocoStubs implements Stubs<ActualHttpServer>```

Reply:
ActualHttpServer is actually not an interface. The mock intention was to hide it, however when loading multiple jsons it was quite easy to do when using that class. so i don't want to expose it, because internal impl of moco might change in future